startDevEnv.sh

```diff
@@ -2,6 +2,5 @@
 
 npm install
 npm run package
```

Comment:
```suggestion
npm run prepare
```

This change is not strictly related to your pr, but seems required since at some point the default npm script name changed and we missed to reflect that here. For simplicity sake I would just add it here.